gen/wpilibj_frcnetcomm.py

```diff
@@ -27,7 +27,7 @@ def main():
         configPath = ".."
 
     outputName = configPath + \
-        "/hal/src/main/java/edu/wpi/first/wpilibj/hal/FRCNetComm.java"
+        "/hal/src/generated/java/FRCNetComm.java"
```

Comment:
I'm not sure why this works as-is, but I really think the package path should be kept for consistency.